bundles/CoreBundle/DependencyInjection/Configuration.php

```diff
@@ -1207,6 +1207,24 @@ private function addSecurityNode(ArrayNodeDefinition $rootNode)
                 ->arrayNode('security')
                     ->addDefaultsIfNotSet()
                     ->children()
+                        ->arrayNode('password')
+                            ->addDefaultsIfNotSet()
+                            ->children()
+                                ->enumNode('algorithm')
+                                    ->info('The hashing algorithm to use for backend users and objects containing a "password" field.')
+                                    ->example('!php/const PASSWORD_BCRYPT')
+                                    ->values([PASSWORD_BCRYPT, PASSWORD_ARGON2I, PASSWORD_ARGON2ID])
```

Comment:
```suggestion
                                    ->values([PASSWORD_DEFAULT, PASSWORD_BCRYPT, PASSWORD_ARGON2I, PASSWORD_ARGON2ID])
```
Should we add `PASSWORD_DEFAULT` as possible value as well, as it could be in theory that it's different from the given values.